commitizen/defaults.py

```diff
@@ -129,3 +135,4 @@ class Settings(TypedDict, total=False):
 bump_message = "bump: version $current_version → $new_version"
 
 commit_parser = r"^((?P<change_type>feat|fix|refactor|perf|BREAKING CHANGE)(?:\((?P<scope>[^()\r\n]*)\)|\()?(?P<breaking>!)?|\w+!):\s(?P<message>.*)?"  # noqa
+version_parser = r"(?P<version>([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?(\w+)?)"
```

Comment:
Not quite sure. but should we use version_scheme or  reuse https://github.com/commitizen-tools/commitizen/blob/master/commitizen/version_schemes.py#L36C1-L36C1?